packages/create-remix/__tests__/create-remix-test.ts

```diff
@@ -242,7 +242,7 @@ describe("create-remix CLI", () => {
     });
 
     expect(stderr.trim()).toMatchInlineSnapshot(
-      `"▲  Oh no! The path \\"this/path/does/not/exist\\" was not found in this GitHub repo."`
```

Comment:
you can reduce the diff from the upgrade if you want by specifying `snapshotFormat`

Reply:
The new defaults are fine imo